Update Readme for contoso-real-estate (#5)
MicrosoftDocs/mslearn-copilot-plugins: contoso-real-estate/README.md

- copilot
- ms-copilot
name: "Contoso real estate plugin for Microsoft Copilot"
description: "A tutorial to learn how to build an API plugin for Microsoft Copilot"
urlFragment: "contoso-real-estate-plugin-for-microsoft-copilot"
---
# Learn to build an API plugin for Microsoft Copilot - Contoso real estate

A tutorial to learn how to build this sample API plugin can be found at [Get started building an API plugin for Microsoft Copilot](https://learn.microsoft.com/copilot-plugins/get-started).

This API plugin finds properties for sale on the market. When the user asks a question about real estate properties, Microsoft Copilot can use this plugin to find properties that have matching criteria. The plugin uses hard-coded data to return results.

## Local development

1.  Clone this repository.
1.  At a command prompt, change to the `contoso-real-estate` directory: `cd contoso-real-estate`.
1.  Run `npm install` to install dependencies.
1.  Start the server with `node server.js`. The server starts on port `8080`.
1.  Test the API, such as by sending a request to `http://localhost:8080/get-listings?city=Paris&bedrooms=3`.

## Testing the plugin

> Sideload the plugin in Bing [bing.com/chat](https://bing.com/chat) by following the instructions in [Test and debug an API plugin](https://learn.microsoft.com/copilot-plugins/test-debug/sideload-test-plugins).

Author: https://www.bing.com/copilot/devtools
instead of bing.com/chat?
@captainbrosset 

to reword, here is some draft text from other PR:
```
To test and debug an API plugin for Microsoft Copilot, you sideload the API plugin 
in Copilot in Bing by using [https://www.bing.com/copilot/devtools](https://www.bing.com/copilot/devtools).
```

Author: ```
To use and test this API plugin in Copilot in Bing, see [Test and debug an API plugin](https://learn.microsoft.com/copilot-plugins/test-debug/sideload-test-plugins).
```

> Sideload the plugin in Bing [bing.com/chat](https://bing.com/chat) by following the instructions in [Test and debug an API plugin](https://learn.microsoft.com/copilot-plugins/test-debug/sideload-test-plugins).

Author: ```suggestion
To use and test this API plugin in Copilot in Bing, see [Test and debug an API plugin](https://learn.microsoft.com/copilot-plugins/test-debug/sideload-test-plugins).
```
